scene/resources/primitive_meshes.cpp

```diff
@@ -3057,10 +3057,10 @@ void TextMesh::_create_mesh_array(Array &p_arr) const {
 				}
 
 				for (int r = 0; r < glyphs[j].repeat; r++) {
-					min_p.x = MIN(gl_data.min_p.x + offset.x, min_p.x);
-					min_p.y = MIN(gl_data.min_p.y - offset.y, min_p.y);
-					max_p.x = MAX(gl_data.max_p.x + offset.x, max_p.x);
-					max_p.y = MAX(gl_data.max_p.y - offset.y, max_p.y);
+					min_p.x = MIN(gl_data.min_p.x + offset.x + gl_of.x, min_p.x);
+					min_p.y = MIN(gl_data.min_p.y - offset.y - gl_of.y, min_p.y);
```

Comment:
I was looking into whether `offset.x + gl_of.x` and `offset.y - gl_of.y` could be stored in a local const to avoid recomputing them all the time (though we can hope the optimizer handles that).

But this seems to be the only place (with `max_p.y`) where that couldn't be used due to a sign difference. I figured I'd flag it in case it's an oversight, or if indeed `gl_of.y` should always be subtracted independently of what's done with `offset.y`.

Reply:
Right, so this confirms my intuition that moving `offset.x + gl_of.x` and `offset.y - gl_of.y` to local constants might make things a bit safer to avoid such mistakes, and make the intent clearer.

Reply:
`offset` is modified for each glyph, this can be done only for `y` in the `glyphs[j].repeat` loops (which is `1` in 99% of cases and unlikely ever exceeds `10`) so should not have any performance impact.